config/ksql-server.properties

```diff
@@ -78,4 +78,4 @@ compression.type=snappy
 #------ Schema Registry -------
 
 # Uncomment and complete the following to enable KSQL's integration to the Confluent Schema Registry:
-# ksql.schema.registry.url=http://localhost:8081
+ksql.schema.registry.url=http://localhost:8081
```

Comment:
This will have to be backed out to merge, right?

Reply:
I missed it. Just updated the PR.